pvlib/clearsky.py

```diff
@@ -151,9 +151,9 @@ def lookup_linke_turbidity(time, latitude, longitude, filepath=None,
     ----------
     time : pandas.DatetimeIndex
 
-    latitude : float
+    latitude : numeric
```

Comment:
`numeric` implies non-scalar input is ok, which is not true here. See https://pvlib-python.readthedocs.io/en/stable/contributing.html#documentation

Reply:
Ok, so I'll change it to float or int. I'll also fix the test argument issue.